src/plugins/status_page/public/status_page.js

```diff
@@ -28,9 +28,11 @@ const chrome = require('ui/chrome')
 
       const data = resp.data;
       ui.metrics = data.metrics;
-      ui.statuses = data.status.statuses;
       ui.name = data.name;
 
+      ui.statuses = _.groupBy(data.status.statuses, s => s.plugin ? 'plugins' : 'generic');
```

Comment:
I would be up for this, but this logic is all front-end, where it's just processing a list of JSON objects